app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -0,0 +1,156 @@
+package org.oppia.app.databinding;
+
+import android.graphics.PorterDuff;
+import android.graphics.drawable.Drawable;
+import android.widget.ImageView;
+
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.databinding.BindingAdapter;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.DataSource;
+import com.bumptech.glide.load.engine.GlideException;
+import com.bumptech.glide.request.RequestListener;
+import com.bumptech.glide.request.RequestOptions;
+import com.bumptech.glide.request.target.Target;
+
+import org.oppia.app.R;
+import org.oppia.app.model.LessonThumbnailGraphic;
+import org.oppia.app.model.ProfileAvatar;
+
+/**
+ * Holds all the custom binding adapters that bind to [ImageView] adapters.
+ */
+public final class ImageViewBindingAdapters {
+  /**
+   * Allows binding drawables to an [ImageView] via "android:src".
+   * Source: https://stackoverflow.com/a/35809319/3689782.
```

Comment:
Here & below: should say 'reference' not 'source'.

Reply:
Okay! :)